src/Raven.Client/Documents/Operations/Backups/BackupStatus.cs

```diff
@@ -50,6 +50,19 @@ public virtual DynamicJsonValue ToJson()
         }
     }
 
+    public class LastRaftIndex : BackupStatus
+    {
+        public long? DatabaseRecord { get; set; }
```

Comment:
`DatabaseRecord` that is nullable `long`? I don't understand what this is meant to do